generated/proto/policy.proto

```diff
@@ -34,3 +34,12 @@ message Policy {
 	Resolution resolution = 1;
 	Retention retention = 2;
 }
+
+message CompressedPolicy {
+	Policy policy = 1;
+	int64 id = 2;
+}
+
+message ActivePolicies {
```

Comment:
Talked offline and agreed that we want to include a cutover time that would be used by the tool which exposes policies to users but the collector will updates its policies immediately because we should only ever add policies.